docusaurus/src/pages/index.js

```diff
@@ -37,40 +43,113 @@ export default function Home() {
 
     return (
         <OpenZitiLayout>
-            <div style={{display:"flex", alignItems:"center", flexDirection:"column"}}>
-                <ThemedImage
-                    alt="OpenZiti Logo"
-                    style={{paddingTop: "75px", paddingBottom: "10px", width:"50%", alignSelf: "center", minWidth: "400px"}}
-                    sources={{
-                        light: useBaseUrl('/img/ziti-logo-dark.svg'),
-                        dark: useBaseUrl('/img/ziti-logo-light.svg'),
-                    }}
-                />
-                <div style={{display:"flex", justifyContent:"center"}}>
-                <p style={{fontSize: "x-large"}}>-- Replacing Infrastructure With Software --</p>
+            <OpenZitiHorizontalSection style={{padding: "0px"}}>
```

Comment:
I know this is probably a "me" thing, but this section of 4 boxes is enormous.  I zoomed to 80% to not feel like I'm in an Accessibility view (which leaves a ton of white space on the sides).  Did you try a version similar to the site we reviewed where the 4 boxes are in a single row?  I know the other site only had 3 boxes and way fewer words in the general description box, so I'm guessing it didn't work...

Reply:
There's way too much text in that top box to do that and as is, the 3 boxes below that just barely fit on a 1080p screen. I made sure to test the design on 1080p and made sure they fit on that page.  The site we reviewed has 3 total 'columns' and far fewer words.

If we want far fewer words, in the boxes - and no icons, it could fit 4 boxes across. also, i had to expand the "max width" to 1300 (from 1200) to accommodate this design as is.  

But - all that is probably something that we can shake out when the extra designer-skillz are applied

Reply:
yeah - we'd want to drop most of the words in the top box if going with 4-across